services/wallet/controllers_v3_test.go

```diff
@@ -306,6 +309,12 @@ func TestLinkBitFlyerWalletV3(t *testing.T) {
 		t.Logf("%s, %+v\n", body, err)
 		must(t, "invalid response", fmt.Errorf("expected %d, got %d", http.StatusOK, resp.StatusCode))
 	}
+
+	var res wallet.LinkDepositAccountResponse
+	err = json.NewDecoder(w.Body).Decode(&res)
+	assert.NoError(t, err)
```

Comment:
Code must not proceed past this check if it fails. Therefore, we must `FailNow` instead of just `Fail`, which in this assertion library is available in the `require` package. Same API, but all failed expectations result in an immediate failure of a test.

```suggestion
	require.NoError(t, err)
```

The actual check below should be `assert`, because there are no further steps that depend on that.